src/components/elements/text/text.tsx

```diff
@@ -89,7 +90,17 @@ export const Text = (props: IGenericElementProps<ITextSchema>) => {
 				// NOTE: Parent text body should be transformed into <div> to prevent validateDOMNesting error
 				{...(hasNestedFields() && { as: "div" })}
 			>
-				<Sanitize inline sanitizeOptions={{ allowedAttributes: false }}>
+				<Sanitize
+					inline
+					sanitizeOptions={{
+						allowedTags: ["img", "br", "p", "span"],
+						allowedAttributes: {
+							img: ["src", "alt", "width", "height"],
+							p: ["class", "data-testid", "id"],
+							span: ["class", "data-testid", "id"],
+						},
+					}}
+				>
```

Comment:
the default allowed tags and attributes are a lot more exhaustive, you can refer to their [docs](https://github.com/apostrophecms/sanitize-html?tab=readme-ov-file#i-like-your-set-but-i-want-to-add-one-more-tag-is-there-a-convenient-way) to see how to extend from their default list

Reply:
follow the doc, use defalut list + img tag